providers/modelcaps/registry.go

```
@@ -0,0 +1,273 @@
+	"slices"
+	"sort"
+	"strings"
+	"sync"
+
+	"github.com/deepnoodle-ai/dive/llm"
+)
+
+// BudgetBounds describes fixed manual reasoning-budget bounds. Nil means Dive
+// has no fixed bound to publish.
+type BudgetBounds struct {
+	noUnkeyedLiterals struct{}
```

golangci-lint (2.13.2)

[error] field noUnkeyedLiterals is unused

(unused)